lib/src/test/java/com/auth0/android/lock/WebProviderTest.java

```diff
@@ -262,14 +215,6 @@ public void shouldResumeWithIntent() {
         assertThat(webProvider.resume(intent), is(false));
     }
 
-    @Test
-    public void shouldResumeWithCodesAndIntent() {
-        Intent intent = mock(Intent.class);
-        Options options = mock(Options.class);
-        WebProvider webProvider = new WebProvider(options);
-        assertThat(webProvider.resume(1, Activity.RESULT_CANCELED, intent), is(false));
```

Comment:
launching with a result code is no longer valid. This was only used for the "webview" flow.